README.md

```diff
@@ -1,36 +1,54 @@
-This is a [Next.js](https://nextjs.org/) project bootstrapped with [`create-next-app`](https://github.com/vercel/next.js/tree/canary/packages/create-next-app).
-
 ## Getting Started
 
-First, run the development server:
+### Step 1: Install MongoDB
 
-```bash
-npm run dev
-# or
-yarn dev
-# or
-pnpm dev
-# or
-bun dev
-```
+1. Go to the [MongoDB Download Center](https://www.mongodb.com/try/download/community).
+2. Select your operating system and download the installer.
+3. Follow the installation instructions for your OS.
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+### Step 2: Install MongoDB Compass (GUI)
 
-You can start editing the page by modifying `app/page.tsx`. The page auto-updates as you edit the file.
+1. Download MongoDB Compass from the [MongoDB Compass Download Page](https://www.mongodb.com/try/download/compass).
+2. Install Compass by following the provided instructions.
 
-This project uses [`next/font`](https://nextjs.org/docs/basic-features/font-optimization) to automatically optimize and load Inter, a custom Google Font.
+### Step 3: Connect to MongoDB using Compass
 
-## Learn More
+1. Open MongoDB Compass.
```

Comment:
Add a note to join the Discord for dev credentials with this link:
https://discord.com/invite/Rvthxry